docs/IE_DG/supported_plugins/GNA.md

```diff
@@ -159,6 +162,29 @@ Heterogeneous plugin was tested with the Intel&reg; GNA as a primary device and
 
 > **NOTE:** Due to limitation of the Intel&reg; GNA backend library, heterogenous support is limited to cases where in the resulted sliced graph, only one subgraph is scheduled to run on GNA\_HW or GNA\_SW devices.
 
+## Recovery from interruption by high-priority Windows audio processes\*
+
+As noted in the introduction, GNA is designed for real-time workloads such as noise reduction or speech recognition. 
+For such workloads, processing should be time constrained, otherwise extra delays may cause undesired effects such as
+audio "glitches". To make sure that processing can satisfy real time requirements, the GNA driver provides a QoS
+(Quality of Service) mechanism which interrupts the requests which cannot be fulfilled in time.
```

Comment:
Not quite right.  Should be "interrupts requests that might cause high-priority Windows audio processes to miss schedule".  Perhaps combine with the next sentence.

Reply:
Fixed